compiler/rustc_const_eval/src/const_eval/eval_queries.rs

```diff
@@ -67,7 +67,7 @@ fn eval_body_using_ecx<'mir, 'tcx>(
     trace!(
         "eval_body_using_ecx: pushing stack frame for global: {}{}",
         with_no_trimmed_paths!(ecx.tcx.def_path_str(cid.instance.def_id())),
-        cid.promoted.map_or_else(String::new, |p| format!("::promoted[{p:?}]"))
+        cid.promoted.map_or_else(String::new, |p| format!("::{p:?}"))
```

Comment:
`p` has type `mir::Promoted` which already [adds](https://github.com/rust-lang/rust/blob/18715c98c67235274843974c928a70e84946398b/compiler/rustc_middle/src/mir/mod.rs#L1605) the `promoted[...]` in its debug format.